### ISSUE TITLE
Accesibility issue in edit/access-control

### ISSUE THREAD

**Issue by CrisGuzmanS** [DSpace/dspace-angular#2775]
**Describe the bug**
In the edit section for access control there are two accessibility errors regarding empty labels.

**To Reproduce**
1. Log in as an admin
2. edit any access-control of a community
3. Open WAVE Evaluation Tool

![image](https://github.com/DSpace/dspace-angular/assets/51240376/39a363a6-d7b8-478a-8f5e-b948bb020e4d)

**Expected behavior**
There should be no error

@tdonohue i can take this activity

**Comment by tdonohue** [comment]
@CrisGuzmanS : Sounds good. I'll assign back to you then and move this to the 7.6.x board.  Please submit a PR when you are ready and we'll get it applied to 8.0 and 7.6.x

**Comment by alexandrevryghem** [comment]
@tdonohue & @CrisGuzmanS: This should already fixed by PR #2717

**Comment by tdonohue** [comment]
@alexandrevryghem : Thanks for catching that.  It's very difficult to keep all these accessibility issues separate :)

@CrisGuzmanS: If you want to test out PR #2717 to see if it's already solved there, that would be appreciated.  If it is solved, then there's no need for you to do any work.  If it is NOT solved, then you can claim this as a separate task

(#2717 is also on my list to test, but I have a very long list right now)

**Comment by CrisGuzmanS** [comment]
Hello @tdonohue & @alexandrevryghem , i have already verified that the issue has been solved in #2717 
![image](https://github.com/DSpace/dspace-angular/assets/51240376/348b64eb-a2a5-48d5-889e-c8942ffea2d8)

**Comment by tdonohue** [comment]
Thanks @CrisGuzmanS for your help in testing that PR!  In that case, I'm going to link this ticket up to #2717 and close the #2776 PR as a duplicate. Apologies again for the confusion.

(I'll try to move #2717 up my list too in order to avoid this happening again, as it seems to fix a lot of small accessibility issues)
